src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -110,6 +111,7 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
         .withPoolingOptions(poolingOptions)
         .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
         .withQueryOptions(new QueryOptions().setFetchSize(fetchSize))
+        .withPort(port)
```

Comment:
Thanks for testing, and for the pointer to ccm. I hadn't come across it yet. That will be really helpful.